cmd/krel/cmd/changelog_test.go

```diff
@@ -96,7 +96,6 @@ func TestNewPatchRelease(t *testing.T) { // nolint: dupl
 		require.Nil(t, err)
 		require.Contains(t, string(changelog), patchReleaseExpectedTOC)
 		require.Contains(t, string(changelog), patchReleaseExpectedContent)
-		require.Contains(t, string(changelog), patchReleaseDeps)
```

Comment:
We still need this check, right?

Reply:
Yes sorry I had left it out when I was debugging. It's back now.